webrecorder/browsertrix
# Edit crawl schedule

(https://github.com/webrecorder/browsertrix-cloud/issues/89) Allow user to edit crawl schedule from crawl template list or detail view.

### Manual testing
1. Run app and go to crawl templates page.
2. Click "..." more icon, choose "Edit crawl schedule"
3. Update schedule. Verify changes save as expected
4. Click on a template and scroll down to schedule. Click "Edit"
5. Update schedule. Verify changes save as expected

### Screenshots
**New menu item:**
<img width="518" alt="Screen Shot 2022-01-25 at 6 15 55 PM" src="https://user-images.githubusercontent.com/4672952/151095033-a686b382-8894-4faf-badc-d6c892116b47.png">

**Modal:**
<img width="1056" alt="Screen Shot 2022-01-25 at 6 16 03 PM" src="https://user-images.githubusercontent.com/4672952/151095053-39d0cd00-cab3-48e1-96ee-8183eed580db.png">

**Edit button in detail page:**
<img width="856" alt="Screen Shot 2022-01-25 at 6 16 12 PM" src="https://user-images.githubusercontent.com/4672952/151095089-6703cb39-52a3-45f6-8533-ede5b32e1330.png">

**Form in detail page:**
<img width="849" alt="Screen Shot 2022-01-25 at 6 16 17 PM" src="https://user-images.githubusercontent.com/4672952/151095110-702020b6-d21f-43d8-b19a-826e97b5726d.png">
